### ISSUE TITLE
RUBYGEMS_GEMDEPS: undefined method `version' for nil:NilClass

### ISSUE THREAD

**Issue by michaelrkn** [rubygems/rubygems#1223]
```
$ git clone https://github.com/emberjs/guides/
$ cd guides/
```

With `RUBYGEMS_GEMDEPS=-`:

```
$ middleman
/Users/michael/.rubies/ruby-2.1.4/lib/ruby/site_ruby/2.1.0/rubygems/request_set/lockfile/parser.rb:324:in `pinned_requirement': undefined method `version' for nil:NilClass (NoMethodError)
	from /Users/michael/.rubies/ruby-2.1.4/lib/ruby/site_ruby/2.1.0/rubygems/request_set/lockfile/parser.rb:80:in `parse_DEPENDENCIES'
	from /Users/michael/.rubies/ruby-2.1.4/lib/ruby/site_ruby/2.1.0/rubygems/request_set/lockfile/parser.rb:22:in `parse'
	from /Users/michael/.rubies/ruby-2.1.4/lib/ruby/site_ruby/2.1.0/rubygems/request_set.rb:282:in `load_gemdeps'
	from /Users/michael/.rubies/ruby-2.1.4/lib/ruby/site_ruby/2.1.0/rubygems.rb:1055:in `use_gemdeps'
	from /Users/michael/.rubies/ruby-2.1.4/lib/ruby/site_ruby/2.1.0/rubygems.rb:1244:in `<top (required)>'
	from <internal:gem_prelude>:1:in `require'
	from <internal:gem_prelude>:1:in `<compiled>'
```

Without:

```
$ bundle exec middleman
git://github.com/LeonB/middleman-swiftype.git (at master) is not yet checked out. Run `bundle install` first.
```

**Comment by aripollak** [comment]
I think this is the same as #1210.

**Comment by indirect** [comment]
This should be fixed in RubyGems 2.5.2. Please upgrade with `gem update --system`, and let us know if things aren't working.
